### PR TITLE
evm: Update NTT token interface

### PR DESCRIPTION
**Pull request by RahulMaganti47**

### CODE REVIEW

File: evm/README.md [hunk 1]
```diff
@@ -244,3 +244,8 @@ bash sh/configure_wormhole_ntt.sh -n NETWORK_TYPE -c CHAIN_NAME -k PRIVATE_KEY
 -n testnet, mainnet
 -c avalanche, ethereum, sepolia
 ```
+
+#### Additional Notes
+Tokens powered by NTT require the `burn` method to be present. This method is not present in the standard ERC20 interface, but is found in the `ERC20Burnable` interface.
```
Comment (gator-boi) [inline evm/README.md:249]: This is only true when the contract is in `burn` mode. NTT can support existing ERC20's without `burn` by using `locking` mode.

File: evm/src/interfaces/INTTToken.sol [hunk 2]
```diff
@@ -2,11 +2,35 @@
...
+    /// @notice the caller is not the minter.
+    /// @dev Selector 0x5fb5729e.
+    /// @param caller The caller of the function.
     error CallerNotMinter(address caller);
+
+    /// @notice the minter is the zero address.
+    /// @dev Selector 0x04a208c7.
     error InvalidMinterZeroAddress();
 
+    /// @notice insufficient balance to burn the amount.
```
Comment (gator-boi) [inline evm/src/interfaces/INTTToken.sol:14]: Capitalize to be consistent with other natspec here.

**Review by gator-boi — APPROVED** [review]

File: evm/README.md [hunk 3]
```diff
@@ -244,3 +244,8 @@ bash sh/configure_wormhole_ntt.sh -n NETWORK_TYPE -c CHAIN_NAME -k PRIVATE_KEY
 -n testnet, mainnet
 -c avalanche, ethereum, sepolia
 ```
+
+#### Additional Notes
+Tokens powered by NTT in __burn__ mode require the `burn` method to be present. This method is not present in the standard ERC20 interface, but is found in the `ERC20Burnable` interface.
+
+The `mint` and `setMinter` methods found in the [`INTTTokenInterface`](src/interfaces/INTTToken.sol) are not found in the standard `ERC20` interface.
```
Comment (nik-suri) [inline evm/README.md:251]: nit -- "`INTTToken` Interface"

**Review by nik-suri — APPROVED** [review]
LGTM -- minor nit.

I'm also wondering whether we should rename `INTTToken` to `INttToken`, which seems more readable and also consistent with `NttManager`

**Review by gator-boi — APPROVED** [review]
lgtm

**Review by nik-suri — APPROVED** [review]

**Pull request merged**